solr/contrib/ltr/src/java/org/apache/solr/ltr/norm/MinMaxNormalizer.java

```diff
@@ -73,6 +74,13 @@ public void setMax(String max) {
     this.max = Float.parseFloat(max);
     updateDelta();
   }
+  
+  @Override
+  protected void validate() throws ModelException {
+    super.validate();
+    if (delta == 0f)
+      throw new ModelException("MinMax Normalizer delta must not be zero");
```

Comment:
I think we should either throw a `NormalizerException` here (the class does not yet exist) or combine the (currently identical) `ltr.feature.FeatureException` and `ltr.model.ModelException` exceptions into one `ltr.LTRException` exception. Alternatively `ltr.norm` could become `ltr.model.norm` and then using `ModelException` here would be less confusing.

My preference would be for `ltr.norm.NormalizerException` to be added since that would provide more clarity than a general LTRException.